### PR TITLE
Update kubectl reference docs for 1.26

### PR DESCRIPTION
**Pull request by tengqm**
This PR updates the kubectl reference using the HTML generator.

### CODE REVIEW

**Comment by sftim** [comment]
/sig release

Sounds good to me.
/approve

**Review by sftim — COMMENTED** [review]
Is the path `static/docs/reference/generated/kubectl/node_modules` right? It feels like an implementation detail we shouldn't expose.

**Comment by tengqm** [comment]
> Is the path `static/docs/reference/generated/kubectl/node_modules` right? It feels like an implementation detail we shouldn't expose.

Thanks for checking. The generator for this reference is using a docker image, which we don't have source code. The generated HTML has the following lines:

```
<!-- Latest compiled and minified CSS -->
<link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
<link rel="stylesheet" href="node_modules/font-awesome/css/font-awesome.min.css" type="text/css">
<link rel="stylesheet" href="node_modules/highlight.js/styles/default.css" type="text/css">
<link rel="stylesheet" href="stylesheet.css" type="text/css">
</head>
```

That is bad. We have to live with it before we get some bandwidth to rework the flow/tool.

File: static/docs/reference/generated/kubectl/node_modules/jquery/dist/jquery.min.js [hunk 1]
```diff
@@ -1,2 +1,2 @@
-/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```
Comment (Zhuzhenghao) [inline static/docs/reference/generated/kubectl/node_modules/jquery/dist/jquery.min.js:1]: In my generating, the version of JQuery changes from v3.6.0 to v3.6.1, but in your generating, it is not the same. I'm curious about this change. My generating is as below.
image

Comment (tengqm) [reply static/docs/reference/generated/kubectl/node_modules/jquery/dist/jquery.min.js:1]: I'm not sure about the reason why this thing gets a version bump.

**Comment by tengqm** [comment]
> We'll need a reviewer who knows Golang, right?

I have plan to rework the generator so that

1. the generator for kubectl reference is completely under kubernetes-sigs/reference-docs repo, and
2. the generated reference is markdown with embedded HTML table where appropriate.

This PR is only to keep the reference fresh till I get cycles to rework it.

**Comment by reylejano** [comment]
imo, we should have the kubectl reference docs up for 1.26 before 1.27 comes around
/lgtm

**Pull request merged**